src/layers/StaticImageLayer.js

```diff
@@ -33,7 +33,27 @@ function scaleBounds({ imageWidth, imageHeight, translate, scale }) {
 export default class StaticImageLayer extends CompositeLayer {
   initializeState() {
     const { loader } = this.props;
-    this.setState({ data: loader.getRaster({ z: 0 }) });
+    this.setState({
+      data: loader.getRaster({ z: 0 }),
+      chunkIndex: loader.chunkIndex
+    });
+  }
+
+  updateState() {
+    const { loader } = this.props;
+    this.state.chunkIndex !== loader.chunkIndex &&
+      this.setState({
+        data: loader.getRaster({ z: 0 }),
+        chunkIndex: loader.chunkIndex
```

Comment:
This is where we would want a universal name of sorts - another option is to expose this is a prop and have it be passed in via `indexName: loader.chunkIndex` for example.  Again, we would need to decide on what `indexName` should be.